src/rebar_prv_dialyzer.erl

```diff
@@ -402,15 +402,11 @@ run_dialyzer(State, Opts, Output) ->
     end.
 
 format_warnings(Output, Warnings) ->
-    Warnings1 = format_warnings(Warnings),
+    Warnings1 = rebar_dialyzer_format:format_warnings(Warnings),
     console_warnings(Warnings1),
     file_warnings(Output, Warnings),
     length(Warnings1).
```

Comment:
length is still correct, but that said it still might be more sensible to use length(warnings)